### PR TITLE
Implement delete command

### PR DESCRIPTION
**Pull request by jamessinmaojun**

### CODE REVIEW

**Review by Joshy837 — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/Messages.java [hunk 1]
```diff
@@ -14,7 +14,7 @@ public class Messages {
 
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
-    public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
+    public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person name provided is invalid";
```
Comment (chiageng) [inline src/main/java/seedu/address/logic/Messages.java:17]: Isnt this should be PERSON_DISPLAYED_NAME to avoid confusion?

**Review by chiageng — COMMENTED** [review]
Overall all looks good to me, can ready for merging

**Review by jannaleong — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/Messages.java (hunk 1 above)
Comment (jamessinmaojun) [reply src/main/java/seedu/address/logic/Messages.java:17]: yea I saw I should've changed I'll change in the next iteration!

**Pull request merged**